background.ts

```diff
@@ -316,6 +347,7 @@ Electron.ipcMain.on('k8s-restart', async() => {
   try {
     switch (k8smanager.state) {
     case K8s.State.STOPPED:
+    case K8s.State.VM_STARTED:
```

Comment:
Why are there VM status changes through this PR. Are they related to kim -> nerdctl? I thought that was already handled in a different issue/PR. Can we pull them out as I think this is what's causing the starting issues.